lib/web/JsonpChunkLoadingRuntimeModule.js

```diff
@@ -486,7 +486,9 @@ class JsonpChunkLoadingRuntimeModule extends RuntimeModule {
 						"",
 						`var chunkLoadingGlobal = ${chunkLoadingGlobalExpr} = ${chunkLoadingGlobalExpr} || [];`,
 						"var parentChunkLoadingFunction = chunkLoadingGlobal.push.bind(chunkLoadingGlobal);",
-						"chunkLoadingGlobal.push = webpackJsonpCallback;"
+						"chunkLoadingGlobal.push = webpackJsonpCallback;",
+						"chunkLoadingGlobal = chunkLoadingGlobal.slice();",
+						"for(var i = 0; i < chunkLoadingGlobal.length; i++) webpackJsonpCallback(chunkLoadingGlobal[i]);"
```

Comment:
a very missing feature. Blocking proper support for code splitting in `loadable-components` as long as right now or main chunk has to be loaded first, or the application will be broken.